src/phoenix/trace/llama_index/callback.py

```diff
@@ -173,6 +179,12 @@ def payload_to_semantic_attributes(
     return attributes
 
 
+_EMBARGOED_EVENT_TYPES = (
```

Comment:
Kinda a strong word - why not just call it INGORE

Reply:
Because they aren't ignored...just not exported. We do work on them but they eventually get absorbed by other spans.